### PR TITLE
Feat/hub catalog filters

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/manager-hub`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? |no
| Tickets          | MANAGER-4661
| License          | BSD 3-Clause

## Description

- Improve filter form display

### CODE REVIEW

File: packages/manager/apps/hub/src/catalog/template.html [hunk 1]
```diff
@@ -1,66 +1,69 @@
...
-            size="m"
-        >
-            <oui-select
-                name="universe"
-                model="$ctrl.selectedUniverses"
-                items="$ctrl.universes"
-                multiple
-                on-change="$ctrl.onUniverseChange()"
+            <oui-field
+                label="{{:: 'manager_hub_catalog_universe' | translate }}"
```
Comment (FredericEspiau) [inline packages/manager/apps/hub/src/catalog/template.html]: `data-`

Comment (FredericEspiau) [reply packages/manager/apps/hub/src/catalog/template.html]: Just kidding 😅

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#10382.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#10382.1 ✘  
* 
  * Build ✘

**Pull request merged**